__tests__/offlineUI.js

```diff
@@ -263,17 +270,31 @@ describe('offline UI', async () => {
         };
       },
       pingBotOnLoad: false,
-      setup: () => new Promise(resolve => {
-        const scriptElement = document.createElement('script');
+      props: WEB_CHAT_PROPS,
+      setup: () =>
+      Promise.all([
+        window.WebChatTest.loadScript('https://unpkg.com/core-js@2.6.3/client/core.min.js'),
+        window.WebChatTest.loadScript('https://unpkg.com/lolex@4.0.1/lolex.js')
+      ]).then(() => {
+        window.WebChatTest.clock = lolex.install();
+      })
+    });
 
-        scriptElement.onload = resolve;
-        scriptElement.setAttribute('src', 'https://unpkg.com/core-js@2.6.3/client/core.min.js');
+    await driver.wait(actionDispatched('DIRECT_LINE/CONNECT_PENDING'), timeouts.directLine);
+    await driver.wait(actionDispatched('DIRECT_LINE/CONNECT_FULFILLED'), timeouts.directLine);
+    await driver.wait(actionDispatched('DIRECT_LINE/CONNECT_PENDING'), timeouts.directLine);
 
-        document.head.appendChild(scriptElement);
-      })
+    await driver.executeScript(() => {
+      window.WebChatTest.clock.tick(400); // "Connecting" will be gone after 400ms, turning into "Network interruption occured"
+      window.WebChatTest.clock.tick(200);
+    });
+
+    // TODO: [P4] Understand why we need to fire tick() using two cross-VM calls
+    //       When we put everything in a single cross-VM call, the last tick has no effect
+    await driver.executeScript(() => {
+      window.WebChatTest.clock.tick(1); // Shortly after 15s, it will show "Network interruption occured."
     });
 
-    await driver.sleep(600);
     const base64PNG = await driver.takeScreenshot();
     expect(base64PNG).toMatchImageSnapshot(imageSnapshotOptions);
```

Comment:
Can you add a new line between `const` and `expect(...)`?

At some points later, I am thinking to collapse these two lines into a single line. Will see.